From 5069842ecd67f4ca050d83a2a89b2aec0b87c841 Mon Sep 17 00:00:00 2001
From: Omkar Parte <rakmoparte@gmail.com>
Date: Tue, 10 Mar 2026 23:48:50 +0530
Subject: [PATCH] rodin: Drop X-Reality Engine Causes Dead Screen

Requires Complete Phone Format if turned on by mistake also
---
 res/values/arrays.xml | 2 --
 1 file changed, 2 deletions(-)

diff --git a/res/values/arrays.xml b/res/values/arrays.xml
index fc95a14c576..509afc37368 100644
--- a/res/values/arrays.xml
+++ b/res/values/arrays.xml
@@ -1701,14 +1701,12 @@
     <!-- Reality Display Engine -->
     <string-array name="display_engine_mode_entries">
         <item>@string/display_engine_default</item>
-        <item>@string/x_reality_engine_mode_title</item>
         <item>@string/vivid_engine_mode_title</item>
         <item>@string/triluminos_engine_mode_title</item>
     </string-array>
 
     <string-array name="display_engine_mode_values" translatable="false">
         <item>0</item>
-        <item>1</item>
         <item>2</item>
         <item>3</item>
     </string-array>
-- 
2.53.0

